website/app/components/doc/page/header/index.hbs

```diff
@@ -14,27 +14,40 @@
       <Doc::Page::Header::NavItem @label="Foundations" @route="foundations" @currentTopRoute={{@currentTopRoute}} />
       <Doc::Page::Header::NavItem @label="Components" @route="components" @currentTopRoute={{@currentTopRoute}} />
       <Doc::Page::Header::NavItem @label="Patterns" @route="patterns" @currentTopRoute={{@currentTopRoute}} />
+      <li class="doc-page-header__nav-item-generic doc-page-header__nav-item--split">
+        <Doc::Page::Header::Search />
+      </li>
       <Doc::Page::Header::NavItem
         @label="Support"
         @route="show"
         @model="about/support"
         @currentTopRoute={{@currentTopRoute}}
-        class="doc-page-header__nav-item--split"
       />
-      <li class="doc-page-header__nav-item-generic"><a
-          href="https://github.com/hashicorp/design-system"
-          target="_blank"
-          rel="noopener noreferrer"
-        ><FlightIcon @name="github" @size="24" @isInlineBlock={{false}} /><span class="sr-only">GitHub</span></a></li>
+      <li class="doc-page-header__nav-item-generic">
+        <a href="https://github.com/hashicorp/design-system" target="_blank" rel="noopener noreferrer">
+          <FlightIcon @name="github" @size="24" @isInlineBlock={{false}} />
+          <span class="doc-sr-only">GitHub</span>
+        </a>
+      </li>
     </ul>
   </nav>
-  <button type="button" class="doc-page-header__burger-menu" {{on "click" @onToggleBurgerMenu}}>
-    {{#if @isSidebarVisibleOnSmallViewport}}
-      <FlightIcon @name="x" />
-      Close
-    {{else}}
-      <FlightIcon @name="menu" />
-      Menu
-    {{/if}}
-  </button>
+  <div class="doc-page-header__mobile-only-menu">
+    <button
```

Comment:
This is a "fake" button that triggers the opening of the autocomplete modal (it's the only way to do it, unfortunately)

Reply:
Would you perhaps want to use similar markup for the mobile only nav items as for the non-mobile ones? IE, use a `nav` vs. a `div` to contain the actions and mark them up as a list. (Or perhaps move them inside the existing `nav` element and then hide and show separate the separate actions for mobile vs. non-mobile?)

Reply:
In relation to the automated a11y test failing on the `.aa-Autocomplete`. I'm wondering why don't we use this button (currently only for mobile) for the desktop version too. The markup is compliant and this way we avoid having to set the axe exception across our tests.

Reply:
@alex-ju in any case we need to have the `.aa-Autocomplete` injected, for the other "trigger" buttons to work. In that case, should the `.aa-Automplete` simply have `display: none` ?